infrastructure/src/main/resources/templates/studenten/ergebnis-details.html

```diff
@@ -21,11 +21,9 @@
 <main th:fragment="bodycontent">
     <!-- Restlichen Inhalt hier einfügen! -->
     <h2 th:text="${fragebogen.veranstaltungsname}">Feebackübersicht</h2>
-    <div class="col mb-4" th:each="textfrage :${textFragen}">
-        <h3 th:text="${textfrage.getFragentext()}"></h3>
-        <div th:each="antwort :${textfrage.getAntworten()}">
-            <h4 th:text="${antwort.antworttext}"></h4>
-        </div>
+    <div class="col mb-4" th:each="frage :${fragebogen.fragen}">
+        <h3 th:text="${frage.getFragentext()}"></h3>
+        <p>Eine Antwort</p>
```

Comment:
No answers are entered here by Thymeleaf

Reply:
That is intentional. The display here makes no sense as it is and fixing it took me down a Refactor Rabid hole, trying to make the Antworten more generic. But that is such a big mess, that I just decided to make it its own Branch. So for now this Text is just intentionally static and tomorrow we can fix the Result display functionality.

Reply:
Ok, then everything is ok